modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala

```diff
@@ -6,276 +6,720 @@ package observe.common
 import clue.GraphQLOperation
 import clue.annotation.GraphQL
 import lucuma.schemas.ObservationDB
-import io.circe.Decoder
-import lucuma.core.math.Angle
+import io.circe.{ Decoder, Encoder }
+import io.circe.generic.auto._
+import lucuma.core.math
+import lucuma.core.enum
+import lucuma.core.model
+import cats.syntax.functor._
 
 import java.time
-// gql: import observe.model.decoders._
+import lucuma.core.model.{ Atom, ExecutionEvent, Observation, Step, Target }
+
+// gql: import lucuma.schemas.decoders._
 // gql: import io.circe.refined._
 
 object ObsQueriesGQL {
-  implicit val posAngleDecoder: Decoder[Angle] = Decoder.instance(
-    _.downField("microarcseconds").as[Long].map(Angle.microarcseconds.reverseGet)
-  )
+
+  // I don't know why, but these implicits prevent several warnings in the generated code
```

Comment:
I don't know why this happens either. The `Gid` instances should be in the implicit context.